### PR TITLE
kernel: bump 4.9 to 4.9.17

### PR DESCRIPTION
**Pull request by Ansuel**
Refresh patch

Checked all specific patch 

Signed-off-by: Ansuel Smith <ansuelsmth@gmail.com>

### CODE REVIEW

**Comment by Ansuel** [comment]
Bump to 4.9.17

File: include/kernel-version.mk [hunk 1]
```diff
@@ -4,11 +4,11 @@ LINUX_RELEASE?=1
 
 LINUX_VERSION-3.18 = .43
 LINUX_VERSION-4.4 = .53
-LINUX_VERSION-4.9 = .14
+LINUX_VERSION-4.9 = .16
 
 LINUX_KERNEL_HASH-3.18.43 = 1236e8123a6ce537d5029232560966feed054ae31776fe8481dd7d18cdd5492c
 LINUX_KERNEL_HASH-4.4.53 = 7efecdd5d47c173fc54bbefa5f78e31782532cfa8dbc0df47473e87e35cdf3e6
-LINUX_KERNEL_HASH-4.9.14 = 534d343f4ca5c55d7214694a2a378897e8ee2adf576563f069da010093b3b640
+LINUX_KERNEL_HASH-4.9.16 = 43b444915cbde67d47b6b614e277c14856a55a63698054298be3a16fe3babae4
```
Comment (wongsyrone) [inline include/kernel-version.mk]: You forgot to change the version number and hash.

Comment (Ansuel) [inline include/kernel-version.mk]: i'm so stupid.... changed the kernel-version in the compile dir but not here ahahahah

**Comment by Ansuel** [comment]
Fixed

**Comment by wongsyrone** [comment]
I'm not sure about the change to generic/patches-4.9/072-bcma-from-4.12.patch, others look good to me.

**Comment by Ansuel** [comment]
@wongsyrone problem with working with an outdated branch 
Just a quick rebase with the master repo and that stange patch disappear (that file was modified upstream but because my branch was outdated it shows old version of that)

**Comment by wongsyrone** [comment]
Why do you drop the hunk in target/linux/mvebu/patches-4.9/003-add_switch_nodes.patch

**Comment by nbd168** [comment]
Obsoleted by fb7ea71c15d2

**Pull request closed without merging**